sam-sql-database/src/agents/sql_database/sql_database_agent_component.py

```diff
@@ -106,6 +106,12 @@
                 "description": "Summary of the database schema if auto_detect_schema is False. Will be used in agent description.",
                 "type": "string",
             },
+            {
+                "name": "query_examples",
+                "required": False,
+                "description": "Natural language to SQL query examples to help the agent understand how to query the database. Format: 'Natural Language Query: SQL Query' pairs, one per line. Will be attached to the schema when auto_detect_schema is False.",
+                "type": "string",
```

Comment:
This format feels brittle to me. If someone happens to put a ':' in their natural language part or in the query itself, then it will break. Either this should be an actual object (array of dictionaries) or it should just be a big string that we paste in as is.